iceoryx_posh/include/iceoryx_posh/capro/service_description.hpp

```diff
@@ -77,7 +77,7 @@ constexpr char ScopeTypeString[][MAX_NUMBER_OF_CHARS] = {"WORLDWIDE", "INTERNAL"
 /// instance and the event id.
 /// In order to support different communication protocols, two types of members exist: integer and string identifiers.
 /// If string IDs are used, the integers are initialized to an invalid number. A class object can be
-/// serialized/deserialized, so it is possible to send the information e.g. over a message queue.
+/// serialized/deserialized, so it is possible to send the information e.g. over a IPC channel.
```

Comment:
The SPDX-License-Identifier: Apache-2.0 is missing and you need to add the Apex.AI Inc copyright header.

Reply:
Please scroll up a bit, there we agreed on __code changes__ :smiley_cat:  So I won't change anything here for now.

@dkroenke can we please write down what demands a header change? =)

Reply:
@elfenpiff there is already a PR which adds the SPDX license identifier to all files :)